README.md

```diff
@@ -1,24 +1,29 @@
-# README
+# Apply for postgraduate teacher training
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This service enables postgraduate candidates to apply for initial teacher
+training.
 
-Things you may want to cover:
+## Initial setup
 
-* Ruby version
+Clone this repo and run `bundle install`.
```

Comment:
```suggestion
Clone this repo and run `bundle install`.

You'll also need to configure a local postgres instance.
```

We could give more detail than this, but I suspect we're dockerizing soon anyway?

Reply:
I'm usually inclined to keep quiet on this sort of thing as our docs will inevitably fall out of date. I agree Docker is soon enough that it doesn't matter anyway!